File: Romulox/client-app/src/models/GameDto.js
var GameDto = {
    id: '',
    name: '',
    platformId: '',
    releaseDate: Date.now(),
    developers: '',
    publishers: '',
    description: '',
    iconImage: '',
    image: '', 
    path: '',
    
    init(game) {
      this.id = game.id;
      this.name = game.name;
      this.platformId = game.platformId;
      this.releaseDate = game.releaseDate;
      this.developers = game.developers;
      this.publishers = game.publishers;
      this.description = game.description;
      this.iconImage = game.iconImage;
      this.image = game.image;
      this.path = game.path.slice(7); //slice 'wwwroot/' from beginning of path
    }  
};

export { GameDto }

File: Romulox.Core/Models/GameDto.cs
using System;

namespace Romulox.Core.Models
{
    public class GameDto
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Path { get; set; }
        public Guid PlatformId { get; set; }
        public DateTime? ReleaseDate { get; set; }
        public string Developers { get; set; }
        public string Publishers { get; set; }
        public string Description { get; set; }
        public string Image { get; set; }
    }
}

